src/Tools/Source/RunTests/Program.cs

```diff
@@ -42,7 +44,9 @@ internal static int Main(string[] args)
                 cts.Cancel();
             };
 
-            return Run(options, cts.Token).GetAwaiter().GetResult();
+            var result = Run(options, cts.Token).GetAwaiter().GetResult();
+            CheckTotalDumpFilesSize();
```

Comment:
And if we fail all unit tests at the same time too? We've sure done that before. :smile:

If we increase the limit to 4 GB or so that might be sufficient -- at that point we'd only be deleting unit test dumps if you had multiple unit test runs crashing.